grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -111,13 +111,28 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
         return invoiceItems?.findAll { it.invoice.isRegularInvoice && !it.inverse }?.sum { it.quantity } ?: 0
     }
 
+    /**
+     * Overall invoiced amount for this adjustment. Can be positive, negative or 0.
+     * */
+    BigDecimal getInvoicedAmount() {
+        return invoiceItems?.findAll { it.invoice?.isRegularInvoice && !it.inverse }?.sum { it.amount } ?: 0
```

Comment:
wasn't it possible to do it in one loop as @alannadolny suggested here? https://github.com/openboxes/openboxes/pull/4841#discussion_r1760874919

Reply:
I guess it was, but we have more of these, so I decided to leave it as is, and potentially to add it to the list of "cleanups/improvements"